### ISSUE TITLE
This repository is amazing! 

### ISSUE THREAD

**Issue by xddun** [SainingZhang/DDAMFN#8]
This is the result of my training, my goodness, the effectiveness is fantastic!
(py310_torch)   Tue Mar 19 #   15:54:01 #   /ssd/src_data/eff_train/DDAMFN-main # python affectnet_train.py
Whole train set size: 414798
Validation set size: 5500
[Epoch 1] Training accuracy: 0.4141. Loss: 3.846. LR 0.000100
[Epoch 1] Validation accuracy:0.4827. Loss:7.671
best_acc:0.4827
Model saved.
[Epoch 2] Training accuracy: 0.5221. Loss: 2.075. LR 0.000060
[Epoch 2] Validation accuracy:0.5035. Loss:5.648
best_acc:0.5035
Model saved.
[Epoch 3] Training accuracy: 0.5471. Loss: 1.810. LR 0.000036
[Epoch 3] Validation accuracy:0.5105. Loss:4.621
best_acc:0.5105
Model saved.
[Epoch 4] Training accuracy: 0.5606. Loss: 1.685. LR 0.000022
[Epoch 4] Validation accuracy:0.5198. Loss:3.784
best_acc:0.5198
Model saved.
[Epoch 5] Training accuracy: 0.5674. Loss: 1.618. LR 0.000013
[Epoch 5] Validation accuracy:0.5220. Loss:3.048
best_acc:0.522
Model saved.

处理数据：
[https://qq742971636.blog.csdn.net/article/details/136841591](https://qq742971636.blog.csdn.net/article/details/136841591)

**Comment by simon20010923** [comment]
Thank you very much! We are working to do better. May I post the link on the front page?

**Comment by xddun** [comment]
Your open-source code is extremely useful, and the model architecture performs admirably for this task. It's truly remarkable to see such strong performance on such a challenging problem. 
Thank you very much for your contributions to open-source. Compared to that, my introduction link is insignificant. Therefore, feel free to use it however you like. You can even select some graphics to include or write a better tutorial for beginners. I feel my writing is quite crude; I just consider it as a record. If it can help even a few people, that would be fantastic.

**Comment by simon20010923** [comment]
Our new version DDAMFN++ is available in the folder https://github.com/simon20010923/DDAMFN/DDAMFN++, which includes new alignment code refer to https://github.com/biubug6/Pytorch_Retinaface and optimized by SAM ([Sharpness aware minimization for efficiently improving generalization](https://arxiv.org/abs/2010.01412)). The trained RAF-DB model is available in DDAMFN++/checkpoints_ver2.0, which is not our best. The affectnet model will soon be available.

**Comment by simon20010923** [comment]
Our new version DDAMFN++ is available in the folder https://github.com/simon20010923/DDAMFN/tree/main/DDAMFN%2B%2B, which includes new alignment code refer to https://github.com/biubug6/Pytorch_Retinaface and optimized by SAM ([Sharpness aware minimization for efficiently improving generalization](https://arxiv.org/abs/2010.01412)). The trained RAF-DB model is available in DDAMFN++/checkpoints_ver2.0, which is not our best. The affectnet model will soon be available.
